projects/vanilla/src/styles/app.scss

```diff
@@ -29,6 +29,8 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
 @import "~@fortawesome/fontawesome-free/scss/regular";
 @import "~@fortawesome/fontawesome-free/scss/solid";
 
+@import "~vis-network/dist/dist/vis-network.min.css";
```

Comment:
Maybe the tilde must be removed as his use is deprecated.

Reply:
done